### PR TITLE
Add some docs on the push duty

### PR DESCRIPTION
**Pull request by magopian**

### CODE REVIEW

**Comment by magopian** [comment]
@andym r?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: python
+sudo: false
+python:
+  - "2.7"
+env:
+  - TOX_ENV=docs
+install:
+  - pip install -rrequirements/docs.txt
+script:
+  - cd docs && make html SPHINXOPTS='-nW'
```
Comment (muffinresearch) [inline .travis.yml]: Woud be nice to turn on link checking for this.

Comment (magopian) [inline .travis.yml]: there's the `-n` here already (http://sphinx-doc.org/invocation.html#cmdoption-sphinx-build-n), were you thinking about something else?

Comment (muffinresearch) [inline .travis.yml]: Yeah I was thinking of linkcheck [1] 

[1] http://sphinx-doc.org/invocation.html#invocation-of-sphinx-build

Comment (magopian) [inline .travis.yml]: Ah, excellent, didn't know about that! Added the target to the make invocation, thanks!

**Comment by andymckay** [comment]
looks great, thanks

**Pull request merged**